src/main/scala/com/raquo/airstream/ownership/Owner.scala

```diff
@@ -32,12 +32,15 @@ trait Owner {
     * You can override it to add custom behaviour.
     * Note: You can rely on this base method being empty.
     */
-  protected[this] def onOwned(subscription: Subscription): Unit = ()
+  protected[this] def onOwned(subscription: Subscription): Unit = {
+    val _ = subscription
+  }
```

Comment:
This method shouldn't have code in it. Can we `@nowarn` this away instead of using the `subscription` param?

Reply:
`@nowarn` is not available on scala `2.12`. I will try to go through https://github.com/scala/scala-collection-compat instead.